docs/eventing/samples/gcp-pubsub-source/README.md

```diff
@@ -25,6 +25,12 @@ source is most useful as a bridge from other GCP services, such as
    kubectl apply --filename https://github.com/knative/eventing-sources/releases/download/v0.4.1/gcppubsub.yaml
    ```
 
+   Until 0.5.0 is released (and the previous URL is updated to point at it), you will need to have a copy of the `knative/eventing-sources` repo locally and from its root:
```

Comment:
@adamharwayne  In theory, this work is for the 0.5 release, so we should be documenting the "state of the world" as it will be in 0.5.

Reply:
adamharwayne  has been retired in favor of @Harwayne.

Should I give fake 0.5.0 URLs? Two lines above the URL has 0.4.1, should I just replace it with 0.5.0?

Reply:
Maybe think of it as not-yet-active URLs, instead of fake. We'll double-check that the URLs match the resources that are actually released when v0.5 is cut.

Reply:
I have tested that this guide works as-is (fb03dde). Should I now change it to point at not-yet-active 0.5.0 URLs and submit?

Reply:
Discussed and decided to submit with the expect 0.5.0 URLs.